R/runNMDS.R

```diff
@@ -150,15 +156,16 @@ setGeneric("getNMDS", function(x, ...) standardGeneric("getNMDS"))
 #' @importFrom MASS isoMDS
 #' @importFrom stats cmdscale
 #' @importFrom vegan vegdist monoMDS
-.calculate_nmds <- function(x, FUN = vegdist,
-                            nmdsFUN = c("isoMDS","monoMDS"),
```

Comment:
Also here it seems that you removed the old parameter without deprecation (that was also issue in other function modification that you made previously)

--> Check that the old parameter name still works, we cannot just drop them since users might still use them

Reply:
old parameter name restored.